log/include/gz/utils/log/SplitSink.hh

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2024 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GZ_UTILS_LOG_SPLITSINK_HH_
+#define GZ_UTILS_LOG_SPLITSINK_HH_
+
+#include <memory>
+#include <string>
+
+#include <gz/utils/config.hh>
+#include <gz/utils/ImplPtr.hh>
+#include <gz/utils/log/Export.hh>
+
+#include <spdlog/details/log_msg.h>
+#include <spdlog/pattern_formatter.h>
+#include <spdlog/sinks/base_sink.h>
+#include <spdlog/sinks/stdout_color_sinks.h>
+
+namespace gz
+{
+namespace utils
+{
+namespace log
+{
+inline namespace GZ_UTILS_LOG_VERSION_NAMESPACE {
```

Comment:
I think should just be `GZ_UTILS_VERSION_NAMESPACE`.

Reply:
Changed in [f33b379](https://github.com/gazebosim/gz-utils/pull/134/commits/f33b37925f1611270b01e4f779ad514122cfeaef).